pkg/iostreams/iostreams.go

```diff
@@ -283,13 +285,29 @@ func (s *IOStreams) StopProgressIndicator() {
 
 func (s *IOStreams) StartAlternateScreenBuffer() {
 	if s.alternateScreenBufferEnabled {
+		s.alternateScreenBufferMu.Lock()
+		defer s.alternateScreenBufferMu.Unlock()
+
 		if _, err := fmt.Fprint(s.Out, "\x1b[?1049h"); err == nil {
 			s.alternateScreenBufferActive = true
+
+			ch := make(chan os.Signal, 1)
+			signal.Notify(ch, os.Interrupt)
```

Comment:
Does this need to explicitly handle SIGINT to catch signals sent to the process or is that already handled by `os.Interrupt`?

Reply:
Yes, that's handled by `os.Interrupt`.